app/app/settings.py

```diff
@@ -68,6 +71,10 @@
     "django.middleware.locale.LocaleMiddleware",
 ]
 
+# Add debug toolbar middleware
+if DEBUG:
+    MIDDLEWARE.append("debug_toolbar.middleware.DebugToolbarMiddleware")
```

Comment:
The documentation warns to add the middleware as early as possible in the list, but here it goes last. Anything unique to our situation that warranted this, or should we put our reasoning in a comment?